.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (Optional, without 'v' e.g. 1.2.3)"
+
+jobs:
+  release:
+    name: Tag and release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
```

Comment:
I think this should be conditional. I.e., you want an automatic bump whenever there is no inputs.tag, right? Like, if the previous release was v1.2.3, and I trigger the workflow without any inputs, then I'm expecting an automatic new version v1.2.4. Otoh, if there is `inputs.tag`, this shall be `false`

Reply:
Maybe I misunderstood this a bit previously. I added a condition where `default_bump: false` if there is a non-empty `inputs.tag`. Otherwise, I kept the default to `patch`. Thanks!

Reply:
Thanks. Won't the `custom_tag` interfere with the automatic bump? Cause if `inputs.tag == ''` then `custom_tag: ''`...is that a problem?

Reply:
No. If `custom_tag: ''`, it is ignored and the automatic bump takes over.